[0m10:47:33.089457 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77d38d43cbf0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77d38d5ca2a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77d38d7f90a0>]}


============================== 10:47:33.097985 | fdcfff54-7514-4f56-a55e-27372d515eb3 ==============================
[0m10:47:33.097985 [info ] [MainThread]: Running with dbt=1.9.0
[0m10:47:33.098570 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'profiles_dir': '/root/.dbt', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init my_project', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:47:33.163615 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/root/.dbt')}
[0m10:47:33.164450 [info ] [MainThread]: Creating dbt configuration folder at 
[0m10:47:33.170233 [debug] [MainThread]: Starter project path: /usr/local/lib/python3.12/site-packages/dbt/include/starter_project
[0m10:47:33.183760 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:47:33.184300 [info ] [MainThread]: Setting up your profile.
[0m10:49:19.290172 [info ] [MainThread]: Profile my_project written to /root/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m10:49:19.293318 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 106.55482, "process_in_blocks": "8", "process_kernel_time": 0.577398, "process_mem_max_rss": "92760", "process_out_blocks": "1544", "process_user_time": 1.457432}
[0m10:49:19.294244 [debug] [MainThread]: Command `dbt init` succeeded at 10:49:19.294065 after 106.56 seconds
[0m10:49:19.295171 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77d38d0e1700>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77d38cf26420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x77d38d0d7560>]}
[0m10:49:19.295902 [debug] [MainThread]: Flushing usage events
[0m10:49:20.608495 [debug] [MainThread]: An error was encountered while trying to flush usage events
